cpp/src/aztec/honk/proof_system/prover.hpp

```diff
@@ -28,23 +28,13 @@ template <typename settings> class Prover {
 
     void compute_wire_commitments();
 
-    void compute_grand_product_polynomial(
-        barretenberg::fr beta = 1); // TODO(Cody): get rid of dangerous default value here
+    void compute_grand_product_polynomial(barretenberg::fr beta, barretenberg::fr gamma);
 
     waffle::plonk_proof& export_proof();
     waffle::plonk_proof& construct_proof();
 
     size_t get_circuit_size() const { return n; }
 
-    void flush_queued_work_items() { queue.flush_queue(); }
-
-    waffle::work_queue::work_item_info get_queued_work_item_info() const { return queue.get_queued_work_item_info(); }
-
-    size_t get_scalar_multiplication_size(const size_t work_item_number) const
-    {
-        return queue.get_scalar_multiplication_size(work_item_number);
-    }
-
```

Comment:
Commented this out and placed it along with the other `work_queue` related declarations.